src/Tutor.Core/Domain/Stakeholders/User.cs

```diff
@@ -3,11 +3,11 @@
 public class User
 {
     public int Id { get; private set; }
-    public string Username { get; private set; }
+    public string Username { get; set; }
```

Comment:
Can this be `internal set`?

Btw, if you didn't already, install ReSharper.

Reply:
It can, changed.